storage/src/main/java/com/tencent/rss/storage/handler/impl/HdfsShuffleReadHandler.java

```diff
@@ -108,18 +107,6 @@ public ShuffleDataResult readShuffleData() {
     return data;
   }
 
-  protected ShuffleIndexResult readShuffleIndex() {
```

Comment:
It seems that you just move this method to another position, don't modify this method. Isn't it?

Reply:
Unify the the order in DataSkippableReadHandler.